test/services/SessionServiceSpec.scala

```diff
@@ -83,6 +83,23 @@ class SessionServiceSpec extends TestSupport with MockUIJourneySessionDataReposi
         }
       }
 
+      "setMultipleMongoData method" should {
+        "return a future boolean value" in {
+          mockRepositoryUpdateData()
+          val testMap: Map[String, String] = Map("key1" -> "value1", "key2" -> "value2")
+          val result: Either[Throwable, Boolean] = TestSessionService.setMultipleMongoData(testMap,
+            JourneyType(Add, SelfEmployment))(headerCarrier, ec).futureValue
+          result shouldBe Right(true)
+        }
+        "return a future left exception" in {
+          mockRepositoryUpdateDataFailure()
+          val testMap: Map[String, String] = Map("key1" -> "value1", "key2" -> "value2")
+          val result: Either[Throwable, Boolean] = TestSessionService.setMultipleMongoData(testMap,
+            JourneyType(Add, SelfEmployment))(headerCarrier, ec).futureValue
+          result.toString shouldBe Left(new Exception("Mongo Save data operation was not acknowledged")).toString
+        }
+      }
+
```

Comment:
I thought of some additional tests you can do :)

mockRepositoryUpdateDataFailure will return isAcknowledged = false no matter what, but we want to test that the whole operation fails if the first updateData passes, and only the second one fails. We can make new mocks in this test file to do this, by specifying the values in the mocks (eg: ArgumentMatchers.eq("key1")